subprojects/docs/src/snippets/base/customExternalTask/groovy/task/build.gradle

```diff
@@ -6,17 +6,23 @@ plugins {
 // tag::external-task-build[]
 }
 
+// tag::gradle-api-dependencies[]
 dependencies {
     implementation gradleApi()
 }
+// end::gradle-api-dependencies[]
 // end::external-task-build[]
 
 repositories {
     mavenCentral()
 }
 
-dependencies {
-    testImplementation 'junit:junit:4.13'
+testing {
+    suites {
+        test {
+            useJUnit()
+        }
+    }
```

Comment:
Unfortunately, I don't think we can do this quite yet. This is an incubating feature and we need to show the stable version for now.